examples/resources/iam_policy/resource_iam_cluster.tf

```diff
@@ -0,0 +1,33 @@
+/*
+ Cluster scoped Role Bindings.
```

Comment:
`Cluster scoped IAM policy.`

Same for all other examples. 

Role Bindings -> IAM policy